includes/payments/class-edd-payment.php

```diff
@@ -1987,14 +1987,6 @@ public function update_status( $status = '' ) {
 					$this->process_pending();
 					break;
 			}
-
-			do_action( 'edd_update_payment_status', $this->ID, $status, $old_status );
```

Comment:
I _think_ we want to bring this back _but_ wrap it in:

```php
if ( 'refunded' === $status ) {
    // code here
}
```

Reason being:

- The order only gets updated if the status is NOT `refunded`. 
- That means, if changing to `refunded`, the new order transition hook will never run.
- But previously this hook _did_ run for all statuses, including `refunded`.
- So in short: we need this back _just_ to handle refunds, then all other statuses are covered by the order update.

Reply:
I did this but put it inside the `refunded` case in the switch statement above.